modules/test/playwright/pages/accessibility-menu-web/AccessibilityMenuPage.ts

```diff
@@ -30,6 +30,10 @@ export class AccessibilityMenuPage {
 		this.underlinedLinksToggle = page.getByLabel('Underlined Links');
 	}
 
+	async isAccessibilityMenuAttached() {
+		return (await this.openAccessibilityMenuButton.count()) === 1;
```

Comment:
This is a nitpick. Could we use `expect().toBeAttached()` here? https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-attached

Reply:
If I use `expect().toBeAttached()` and there is no accessibility menu attached, I will get an error. This is for conditional purposes where, as a test scenario setup, it requires the accessibility menu be present or not.